docs/src/main/asciidoc/includes/devtools/prerequisites.adoc

```diff
@@ -16,7 +16,7 @@ ifdef::prerequisites-docker[]
 * A working container runtime (Docker or Podman)
 endif::[]
 ifdef::prerequisites-docker-compose[]
-* Docker and Docker Compose
+* Docker and Docker Compose or xref:podman.adoc[Podman]
```

Comment:
A bit further you ask to install Podman and Docker Compose. If you still need Docker Compose with Podman, I would rather phrase it: `Docker or xref:podman.adoc[Podman], and Docker Compose`